sql/analyzer/indexes.go

```diff
@@ -240,6 +241,10 @@ func getIndexes(
 		result := multiColumnIndexes
 		// Next try to match the remaining expressions individually
 		for _, e := range unusedExprs {
+			// TODO: eventually support merging spatial lookups
+			if _, ok := e.(*spatial.Intersects); ok {
```

Comment:
could maybe use an interface for checking if an expression is a spatial function